### ISSUE TITLE
Need to be clear about users having control, then sites, then browser

### ISSUE THREAD

**Issue by BenjamP** [w3c/editing#13]
Accessibility folks in particular want to make sure that a user has the final say on what Actions indicate a given Intention. So while a site should be able to change the default Intention for an Action, a user should be able to overwrite it with their own Intention for a given Action. This may not need to be in the spec, but it should not be blocked.

**Comment by johanneswilm** [comment]
Given that we no longer support custom intentions, and most things are handled through normal JavaScript, the responsibility for making sure this works also goes there. For the remaining intentions, these are standard UA actions and will be configurable through it as expected. I don't think there is any need to specify this further. Please reopen if disagree.
